examples/abitest/tests/Cargo.toml

```diff
@@ -15,6 +15,9 @@ log = "*"
 oak = "=0.1.0"
 oak_abi = "=0.1.0"
 prost = "*"
+# Using an old version that is supported by `cargo-raze`:
+# https://github.com/google/cargo-raze/issues/41#issuecomment-592274128
+tonic = { version = "=0.1.1", features = ["tls"] }
```

Comment:
I think after https://github.com/project-oak/oak/pull/1016 we can use the latest version of `tonic`

Reply:
Done.